recipes/hangar/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "hangar" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 85bbb1e8fe0cac56ca86257283a96472bd8410443d1b819d0e6b67f0a0d63214
+
+build:
+  number: 1
+  skip: true  # [py<36]
```

This looks like it could be noarch?
```suggestion
  noarch: python
```

---

Oops I forgot to delete this comment, it can't be `noarch` due to the Windows selector.

---

This project does not support anything less than `python 3.6`, (we use language features which aren't back ported to `2.7` or `3.4` / `3.5`).

Correct me if I'm wrong, but it's my understanding that `noarch` refers to a package which can be run on any system, and any python version? Since we don't support some python versions, It can't be `noarch`?

---

`noarch` means the same "build" can be used with multiple platforms and python versions. You can introduce a constraint on the Python version without by putting `python >=3.6` and conda won't try to install it.

In general pure Python packages can be noarch but anything with compiled code can't as different binary formats are used on different platforms and the Python ABI changes between versions.

That said, a limitation of the current conda package format is that `noarch` packages have to always have the same dependencies so the `blosc-hdf5-plugin  # [not win]` line means this can't be made `noarch`.